### PR TITLE
Python 3.12 support

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: src/dockerflow/logging.py [hunk 1]
```diff
@@ -65,6 +65,7 @@ class JsonLogFormatter(logging.Formatter):
             "processName",
             "relativeCreated",
             "stack_info",
+            "taskName",
```
Comment (grahamalama) [inline src/dockerflow/logging.py:68]: Is this a drive-by, or specifically related to 3.12 somehow?

**Review by grahamalama — APPROVED** [review]

File: src/dockerflow/logging.py (hunk 1 above)
Comment (leplatrem) [reply src/dockerflow/logging.py:68]: This is specific, python 3.12 adds this fields and we don't want it in the mozlog fields
